---
title: Research  
layout: default
group: Research
---

<div class="row">

# Integrative structure determination of assemblies 

Structures of several large protein complexes and assemblies are difficult to obtain using a single experimental or computational method. Integrative structure determination fills this gap; various types of experimental data are combined along with principles from physics, statistical inference, and prior models to obtain the structure. The different sources of input information may span multiple scales; for example, X-ray data is at the atomic scale, while FRET distances are at the domain scale. However, these sources can be complementary ; for example, EM maps may provide the shape of a complex while chemical crosslinks may provide the orientation of binding interfaces. We have used structural, biochemical, biophysical, cell biological, genetic, and in-silico bioinformatics information for deducing the structure of assemblies. 

This approach has several advantages. First, the models are more complete than those generated by other methods, since proteins can be modelled at full-length, including regions of unknown structure. Second, different types of information at different scales can be combined objectively, considering the uncertainty of each experiment, and without using arbitrary weights, via the Bayesian inference framework. Third, the approach produces all the models consistent with input information, allowing us to quantity the error bars (precision) of the structure. Finally, models generated by the integrative approach are validated by several methods, including by experimental methods, providing a high level of confidence to the determined structure. 

In contrast to other approaches to structure determination, our specific focus is on determining domain-level organisation of large assemblies, based on medium-resolution (~5-40 A) experimental data. Our models are usually coarse-grained, i.e., represented at worse than atomic resolution, therefore the focus is more on identifying overall domain-level organisation instead of the finer atomic details. Second, since the data can be noisy, ambiguous, sparse, and incoherent (i.e., based on a heterogeneous sample), more than one model can fit the data and the goal to obtain the ensemble of all models that are consistent with the data.  

## Assemblies characterized 
Opportunities exist in our group to work on several such assemblies. Three broad areas include (a) assemblies involved in regulating gene expression, including chromatin remodelers (b) assemblies at cell-cell junctions, and (c) cellular machineries involved in cytoskeleton organization and cell division such as centriolar and centrosomal protein complexes. We collaborate closely with other cell and structural biologists to validate predictions from our models and generate data for the modeling.

Examples. 

## Method development 

Our other focus is in developing rigorous methods and software for computational modeling of protein organization. These methods make structure determination more accurate and efficient by improving upon approaches that are ad hoc, semi-automated, based on trial-and-error, and/or require manual expert intervention. For this purpose, we are inspired by algorithms from the fields of computational statistics, statistical physics, machine learning and statistical inference, optimization, computer vision, and graph theory.

Examples 


### Integrative structures of chromatin-modifying assemblies

<div class="col-md-7 order-md-1">

We recently determined the structures of sub-complexes of the Nucleosome Remodeling and Deacetylase (NuRD) complex, a chromatin-modifying assembly that regulates gene expression and DNA damage repair. It is conserved across plant and animal species and expressed in most metazoan tissues. However, its structure is hard to characterize experimentally. Using Bayesian integrative structure determination, we combined information from published SEC-MALLS, DIA-MS, XLMS, negative stain EM, X-ray crystallography, and NMR spectroscopy, secondary structure and homology predictions. The integrative structures were corroborated by independent cryo-EM maps, biochemical assays, and known cancer-associated mutations.


</div>
<div class="col-md-5 order-md-2 align-self-center">
<img class="img-fluid" src="nude.png" alt="qFit">
<!-- <a href="http://www.ucsf.edu"><img class="inline-block navb-icon" src="/static/img/ucsf_logo_white.svg" alt="University of California, San Francisco (UCSF) logo"></a> -->
We are applying similar methods to study assemblies at cell-cell junctions, cytoskeletal, and centriolar assemblies.

</div>
</div>
<div class="row">

##  Integrative modeling methods

### 1. StOP:

<div class="col-md-7 order-md-1 ">

We developed a method to optimize the sampling-related parameters for modeling assemblies in [**IMP**](https://integrativemodeling.org). [**StOP: Stochastic Optimization of Parameters**](https://www.mdpi.com/2075-1729/11/11/1183) automates the tuning of MCMC parameters such as rigid body and bead move sizes, restraint weights, and replica exchange temperatures. 

</div>

<div class="col-md-3 order-md-2 align-self-center">

<img class="img-fluid" src="stop.png" alt="StOP">
</div>
</div>
<div class="row">

### 2. PrISM:

<div class="col-md-7 order-md-2">

[**PrISM**](https://www.biorxiv.org/content/10.1101/2021.06.22.449385v4) is our recently developed method to identify high and low precision regions in an ensemble of integrative models of large macromolecular assemblies. .

</div>

<div class="col-md-5 order-md-1 align-self-center">
<img class="img-fluid" src="prism.png" alt="prism">
This is part of our effort to develop protocols for validating integrative models deposited in the [**wwPDB (worldwide Protein Data Bank)**](http://www.wwpdb.org/)
</div>
</div>
